packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -669,7 +669,21 @@ flowchart LR
     C -->|Two| E[Result two]
 ```
 
-## Configuration...
+## Configuration
+
+### Renderer
+
+The layout of the flowchart diagram can be don in multiple ways. The default rendering is done using an algorithm called dagre. It is possible to use another algorithm called **elk** for rendering flowcharts which is better for large and more complex diagrams. This is an expermental feature but worth exploring.
```

Comment:
_Suggested changes:_


The layout of the diagram is done with the _renderer._  The default renderer is **dagre.** 

Starting with Mermaid version 9.4, you can use an alternate renderer named **elk**.   The _elk_ renderer is better for larger and/or more complex diagrams. 
```note
The _elk_ renderer is an experimenal feature.
```

You can change the renderer to _elk_ by adding this directive: